src/Artsy/Router/__tests__/buildClientApp.test.tsx

```diff
@@ -6,6 +6,16 @@ import { mount } from "enzyme"
 import React from "react"
 import { graphql } from "react-relay"
 
+jest.mock("Components/NavBar", () => ({
+  NavBar: () => <div />,
+}))
+
+jest.mock("react-relay", () => ({
+  ReactRelayContext: {
+    Provider: ({ children }) => children,
+  },
+}))
```

Comment:
The use of `getDerivedStateFromError` revealed that we were masking a test down below with an unrelated error